bin/is-website-vulnerable.js

```diff
@@ -1,48 +1,35 @@
 #!/usr/bin/env node
 /* eslint-disable no-process-exit */
-'use strict'
-
+/* eslint no-console: ["error", { allow: ["error", "log"] }] */
 const os = require('os')
 const debug = require('debug')('is-website-vulnerable')
 const argv = require('yargs').argv
-const { prompt } = require('enquirer')
-const isUrl = require('is-url-superb')
 const { Audit, RenderConsole, RenderJson, Utils } = require('../index')
+const getUrl = require('./get-url')
 
-let url = process.argv[2]
-debug(`received url argument: ${url}`)
-
-const isWindows = os.type() === 'Windows_NT'
-const isJson = !!argv.json
-const showProgressBar = !isJson && !isWindows
+const getStatus = () => {
```

Comment:
Good job here!
How about we rename it to something more descriptive, maybe like `getEnvironment` or `detectEnvironment`? I think the latter makes more sense but leaving it up to you if those or other naming is better

Reply:
Ok. I agree with you